Dockerfile

```diff
@@ -1,3 +1,13 @@
+## Build iPXE w/ IPv6 Support
+FROM docker.io/centos:centos7 AS builder
+RUN yum install -y gcc git make genisoimage xz-devel
+WORKDIR /tmp
+COPY . .
+RUN git clone --depth 1 http://git.ipxe.org/ipxe.git && \
```

Comment:
Maybe checkout a specific git hash (e.g. the today's HEAD), so that we don't get different code each time?